code/modules/mob/living/carbon/human/say.dm

```diff
@@ -257,27 +106,27 @@
 		verb = speaking.get_spoken_verb(ending)
 	else
 		if(ending == "!")
-			verb = "exclaims"
-		else if(ending == "?")
-			verb = "asks"
+			return pick("exclaims", "shouts", "yells")

```

Comment:
Yeah, and there was a few we adjusted---I think this was one of them, because "!" doesn't automatically equate to yelling. *shrug*. Not a super big deal, TBH.